app/discovery/events.go

```diff
@@ -35,11 +36,18 @@ type DockerClient interface {
 var reGroup = regexp.MustCompile(`/(.*?)/`)
 
 // NewEventNotif makes EventNotif publishing all changes to eventsCh
-func NewEventNotif(dockerClient DockerClient, excludes ...string) (*EventNotif, error) {
-	log.Printf("[DEBUG] create events notif, excludes: %+v", excludes)
+func NewEventNotif(dockerClient DockerClient, excludes, includes []string) (*EventNotif, error) {
+	if excludes == nil {
```

Comment:
why this check and assignment even needed? nil slice acts as an empty slice anyway. Unless I'm missing smth it can be removed

Reply:
fixed, didn't know that `len(nil)` and `range nil` works well and does not cause panic/errors